executor/transaction_processor.go

```diff
@@ -167,6 +167,7 @@ func MakeTxProcessor(cfg *utils.Config) (*TxProcessor, error) {
 		if evm == nil {
 			available := maps.Keys(tosca.GetAllRegisteredProcessorFactories())
 			available = append(available, "aida")
+			available = append(available, "aida-geth")
```

Comment:
I dont think this does anything and I also think its unrelated to this PR.

Reply:
It is unrelated to this PR. 

This is to correct the oversight when `aida-geth` was added, and I thought it was too small to be its own PR. Will refactor.

Reply:
oh this is for logging whats available. Nice catch